packages/generator-volto/README.md

```diff
@@ -23,7 +23,7 @@ See [`volto-generator` compatibility with Volto](https://6.docs.plone.org/volto/
 npm init yo @plone/volto
 ```
 
-This is the shortcut for using `npm init` command. It uses Yeoman (`yo`) and `@plone/generator-volto` and execute them without having to be installed globally.
+This is the shortcut for using `npm init` command. It uses Yeoman (`yo`) and `@plone/generator-volto` and `npm` executes the commands without having the need for the packages to be installed globally.
```

Comment:
npm or pnpm?

```suggestion
This is the shortcut for using `npm init` command.
It uses Yeoman (`yo`) and `@plone/generator-volto`, and `npm` executes the commands without having the need for the packages to be installed globally.
```

Reply:
@stevepiercy npm is correct, pnpm is used only for the development of volto core.
When you install the volto projects it's still npm only and yarn to run :) 
Personally, I would remove this entire section because before it we mentioned commands
to install both items globally so why would we confuse beginners to then mention oh
you can use npm init without the need of what we've asked you todo previously?

Reply:
I agree. A README should be a linear process. Extra stuff should go into documentation, or at the very end of the README if there are no docs, as an addendum.